cocos/ui/UIWebViewImpl-ios.mm

```diff
@@ -124,7 +124,7 @@ - (void)dealloc {
 
 - (void)setupWebView {
     if (!self.wkWebView) {
-        self.wkWebView = [[[WKWebView alloc] init] autorelease];
+        self.wkWebView = [[[[WKWebView alloc] init] autorelease] retain];
```

Comment:
Just change to `[[WKWebView alloc] init]` is enough.

Reply:
Yes that should do it. I will check once and circle back.

Reply:
It works! Tested it on the iOS 13.1.3. I will test it on iOS 9.3.5 in a couple of hours and report back (don't have the device with me right now).

Have added the new commit to the pull request. The end result is, removing the `autorelease` from the `WKWebView` instantiation.